deployments/apiv2/services/users/src/main.py

```diff
@@ -520,25 +541,30 @@ async def register_user(
 
 
 @app.get("/email", status_code=status.HTTP_204_NO_CONTENT)
-async def email_user(
-    request: Request, email: EmailStr = Query(None), type: str = Query(None), user: bool = Query(None)
+async def email_account(
+    request: Request,
+    email: EmailStr = Query(None),
+    id: str = Query(None),
+    type: str = Query(None),
+    user: bool = Query(None),
 ):
-    """Send or resend user account emails.
+    """Send or resend account emails.
 
     No token required for request. Currently sending reset password and new registration emails based on query type.
     """
     # EmailRegistrationError will be raised if random type param is added that isn't verify or reset
     try:
         async with request.state.pgpool.acquire() as con:
             query = (
-                "SELECT id, customer_id, name FROM users WHERE email=$1"
+                "SELECT id, customer_id, name FROM users WHERE email=$1 AND id=$2"
```

Comment:
Why do we need to check both the email and ID? Couldn't we just check one or the other?

Reply:
It's to check that the email the user enters matches the customer id in jwt token sent in the initial email. We don't want them entering the wrong email or someone elses email